kuka_kr16_support/urdf/kr16r1610cybertech_macro.xacro

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="kuka_kr16r1610cybertech" params="prefix">
```

Comment:
I like this macro name. Appending `cybertech` as a sort-of modifier seems to work.

Suggestion though: let's add an underscore between `r1610` and `cybertech` to make it easier to read:

```suggestion
  <xacro:macro name="kuka_kr16r1610_cybertech" params="prefix">
```

the `kr16r1610` is already quite long (and not a natural word, so requires conscious parsing). Adding the underscore clearly separates the family modifier from the variant specification.

Reply:
If you agree, we'd have to update naming of files and the name in other places as well of course.

Reply:
I'm a big fan of improved readability but I was aiming for consistency with `kuka_kr6r700sixx` and similar

Reply:
Hm, true. That would be consistent.

It's just unfortunate that `sixx` is so much easier to recognise / parse after the range modifier.

Let's keep what you have now and see whether it works.